HW04/Vladislav_Mitin/lib/student.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Student
+  attr_reader :name, :surname, :homeworks, :notifications
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @notifications = []
+  end
+
+  def mark_as_read!
+    @notifications = []
+  end
+
+  def to_work!(homework)
+    notification = Notification.new(homework: homework)
+    homework.mentor.notifications << notification
```

Comment:
Maybe we can create a new interface on `Homework` class?
```suggestion
    homework.notify_mentor(notification)
```